features/step_definitions/navigation_steps.rb

```diff
@@ -130,6 +130,17 @@ def find_record_for(object, schema, name)
   click_link(link)
 end
 
+When /^I click "(.*)" (.*) link$/ do |link, position|
```

Comment:
@tansaku , I had a feeling we could have improved on the previous step regular expression to accommodate this little change introduced
``` cucumber
 When /^I click "(.*)"$/ do |link|
   click_link(link)
 end
```
Looks like having both is not DRY enough